dts/bindings/ethernet/atmel,sam-gmac.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2020 Stephanos Ioannidis <root@stephanos.io>
+# SPDX-License-Identifier: Apache-2.0
+
+description: Atmel SAM-family GMAC Ethernet
+
+compatible: "atmel,sam-gmac"
+
+include: ethernet.yaml
+
+properties:
+    reg:
+      required: true
+
+    queue-num:
```

Comment:
Would be better as `num-queues`.  Is you have it, makes me think the property is to select a queue number to use.

Reply:
Used the name `queue-num` here to be consistent with the naming convention used by the GMAC driver, but sure, this can be somewhat misleading.

Will change it to `num-queues`.